storage/ndb/src/mgmsrv/MgmtSrvr.hpp

```diff
@@ -566,7 +566,7 @@ class MgmtSrvr {
   /**
    *   This function is called from "outside" of MgmtSrvr
    *   when a signal is sent to MgmtSrvr.
-   *   @param  mgmtSrvr: The MgmtSrvr object which shall recieve the signal.
+   *   @param  mgmtSrvr: The MgmtSrvr object which shall receive the signal.
```

Comment:
I think it should be "should" instead of "shall" here. Otherwise it looks good! :+1: